### PR TITLE
TM-4283, add AIM link to AIH page

### PR DESCRIPTION
**Pull request by M00staff**
Pretty straight forward - 2 questions:

- Since we have the nested CSS structure, the CSS for this is duplicated... not a huge issue at all, but worth noting
- I'm not sure what that number `pmi-official-item-number` signifies - is it OK to show on the AIH page?

### CODE REVIEW

**Comment by scott062** [comment]
PMI official item number is the # it is during the meeting so everyone attending can keep track of it. Its generated when an audit is run on all the items attached to the meeting. The # is only unique in the meeting. This means its possible to have that # appearing multiple times in AIH which is not helpful - and could confuse users

**Comment by scott062** [comment]
I'd make it so you can see the icon link but not the # for AIH. Don't forget about card too

**Comment by M00staff** [comment]
> I'd make it so you can see the icon link but not the # for AIH. Don't forget about card too

what do we mean by "Don't forget the card too," here?

File: src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx [hunk 1]
```diff
@@ -71,6 +75,15 @@ const AgendaItemCard = props => {
...
             <div className={`status-tag agenda-tag--${agendaStatus}`}>
+              {
+                showAgendaItemMaintenance &&
+              <Link
+                className="ai-id-card-link"
+                to={`/profile/${userRole}/createagendaitem/${perdet$}/${agenda?.id}`}
+              >
+                <FA name="sticky-note" />
+              </Link>
+              }
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx:86]: we'll want the sticky note to show regardless of the link to it - at least i think this is how it is currently set up elsewhere - to test flip the FF or just flip the FF boolean in the component

**Comment by elizabeth-jimenez** [comment]
"Since we have the nested CSS structure, the CSS for this is duplicated... not a huge issue at all, but worth noting", addressed that here https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2557

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**